cluster-provision/okd/scripts/provision.sh

```diff
@@ -231,12 +247,14 @@ spec:
     matchLabels:
       custom-kubelet: cpumanager-enabled
   kubeletConfig:
+     apiVersion: kubelet.config.k8s.io/v1beta1
+     kind: KubeletConfiguration
      cpuManagerPolicy: static
      cpuManagerReconcilePeriod: 5s
 EOF
 
-oc -n openshift-machine-config-operator wait machineconfigpools worker --for condition=Updating --timeout=1800s
-oc -n openshift-machine-config-operator wait machineconfigpools worker --for condition=Updated --timeout=1800s
+timeout 1800s bash -c "until oc -n openshift-machine-config-operator wait machineconfigpools worker --for condition=Updating --timeout=1800s; do sleep 3; done"
```

Comment:
`30m` is more readable for someone with bad maths

Reply:
/me grabbing calculator to double check